react/DocProp.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react'
```

Comment:
Follow the order:
1. Lib imports (react)
2. App imports (vtex.styleguide)
3. Local imports (./components)